resources/shaders/rt_hybrid.hlsl

```diff
@@ -120,16 +127,45 @@ float3 unpack_position(float2 uv, float depth)
 	return (wpos.xyz / wpos.w).xyz;
 }
 
-float3 DoReflection(float3 wpos, float3 V, float3 normal, uint rand_seed)
+float3 DoReflection(float3 wpos, float3 V, float3 normal, uint rand_seed, uint depth)
 {
 	// Calculate ray info
 	float3 reflected = reflect(-V, normal);
 
 	// Shoot reflection ray
-	float3 reflection = TraceReflectionRay(wpos, normal, reflected, rand_seed);
+	float3 reflection = TraceReflectionRay(wpos, normal, reflected, rand_seed, depth);
 	return reflection;
 }
 
+float DoShadowAllLights(float3 wpos, uint depth, inout float rand_seed)
```

Comment:
It was already in the branch from feature_hybrid_to_composition

Reply:
See the other comment on d3d12_deferred_composition_task.hpp for more information about the change. 

If I understand correctly, you're talking about the function DoShadowAllLights? This has been moved here because the implementation is slightly different. This function goes over all lights and returns a factor. The shade_light functions in lighting.hlsl also shade the pixel, which I don't need for shadows.

I could move the code to lighting.hlsl or shadow_ray.hlsl if that would satisfy your comment. Otherwise, we'll have to discuss this further somewhere else.

Reply:
You probably should move it. Otherwise we have lighting code all over the place again. which makes it hard to keep consistent results over different passes.